### PR TITLE
docs: improve installation

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
- add initial configuration in Installation pages
- add note about baseURL and debug toolbar
- small fixes

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: user_guide_src/source/installation/installing_composer.rst [hunk 1]
```diff
@@ -52,7 +52,11 @@ A sample such installation command, using the default project-root "appstarter":
 
     > composer create-project codeigniter4/appstarter --no-dev
 
-After installation you should follow the steps in the "Upgrading" section.
+Initial Configuration
+---------------------
+
+After installation, a few initial configuration is required.
+See :ref:`initial-configuration` for the detail.
```
Comment (MGatner) [inline user_guide_src/source/installation/installing_composer.rst:59]:
```suggestion
After installation, a few initial configurations are required.
See :ref:`initial-configuration` for the details. 
```

File: user_guide_src/source/installation/installing_composer.rst [hunk 2]
```diff
@@ -148,6 +155,12 @@ Copy the ``env``, ``phpunit.xml.dist`` and ``spark`` files, from
 You will have to adjust the system path to refer to the vendor one, e.g., ``ROOTPATH . '/vendor/codeigniter4/framework/system'``,
 - the ``$systemDirectory`` variable in **app/Config/Paths.php**
 
+Initial Configuration
+---------------------
+
+A few initial configuration is required.
+See :ref:`initial-configuration` for the detail.
```
Comment (MGatner) [inline user_guide_src/source/installation/installing_composer.rst:162]:
```suggestion
A few initial configurations are required.
See :ref:`initial-configuration` for the details.
```

File: user_guide_src/source/installation/installing_manual.rst [hunk 3]
```diff
@@ -26,32 +26,33 @@ and extract it to become your project root.
     The symbols that can be used are ``/``, ``_``, ``.``, ``:``, ``\`` and space.
     So if you install CodeIgniter under the folder that contains the special characters like ``(``, ``)``, etc., CodeIgniter won't work.
 
-Setting Up
-----------
+Initial Configuration
+=====================
 
-None
+After installation, a few initial configuration is required.
```
Comment (MGatner) [inline user_guide_src/source/installation/installing_manual.rst:32]:
```suggestion
After installation, a few initial configurations are required.
```

**Review by MGatner — CHANGES_REQUESTED** [review]
Looks good! A few tweaks.

**Comment by kenjis** [comment]
@MGatner Thank you for proofreading!

**Comment by MGatner** [comment]
Anytime! I know I've said it before: your English is incredibly strong for a non-native speaker! 

Most of these corrections aren't actual problems so much as native phrasing, so if you ever get tired of my nitpicks then just tell me to stop 😅

**Review by MGatner — APPROVED** [review]

**Comment by kenjis** [comment]
No problem. Proofreading is welcome!
In fact we have only two active proofreaders now, you and Paul.

**Pull request merged**